benchmark/main.cpp

```diff
@@ -5,6 +5,52 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 // clang-format on
+#include "utils.h"
+
 #include <benchmark/benchmark.h>
+#include <cuda_runtime.h>
+
+#if defined(__linux__)
+#include <unistd.h>
+#endif
+
+namespace {
+
+std::string getHostName() {
+#if defined(__linux__)
+  constexpr int len = 128;
```

Comment:
There is a `HOST_NAME_MAX` in /usr/include/x86_64-linux-gnu/bits/local_lim.h

Reply:
Looks like the macro depends on `__USE_POSIX`. I'll leave it as this doesn't need to be precise.